satpy/tests/reader_tests/test_fci_l1c_nc.py

```diff
@@ -592,3 +592,28 @@ def test_handling_bad_data_vis(self, reader_configs, caplog):
                 name="vis_04",
                 calibration="reflectance")], pad_data=False)
             assert "cannot produce reflectance" in caplog.text
+
+
+class MockNetCDF4FileHandler(object):
+    """Mocked base file handler NetCDF4."""
+
+    def __init__(self, filename, filename_info, filetype_info, cache_var_size=10000, cache_handle=True):
+        """Initialize."""
+        pass
+
+
+class TestFCIL1cNCReaderMissingHdf5plugin(object):
+    """Test FCI L1c NetCDF reader initialization when `hdf5plugin` isn't installed."""
+
+    def test_init(self, reader_configs):
+        """Test file pattern matching."""
+        filename = ("W_XX-EUMETSAT-Darmstadt,IMG+SAT,MTI1+FCI-1C-RRAD-FDHSI-FD--"
+                    "CHK-BODY---NC4E_C_EUMT_20130804121330_GTT_DEV_"
+                    "20130804120815_20130804120830_N_JLS_T_0073_0034.nc")
+        filename_info = {'special_compression': 'JLS'}
+        with mock.patch("satpy.readers.netcdf_utils.NetCDF4FileHandler", new=MockNetCDF4FileHandler):
+            with mock.patch("satpy.readers.fci_l1c_nc.hdf5plugin", new=None):
+                from satpy.readers.fci_l1c_nc import FCIL1cNCFileHandler
+                with pytest.raises(ImportError) as err:
```

Comment:
I see that hdf5plugin is added to the ci environment, but is it needed? since it is mocked here anyway?

Reply:
Good point. That's what you get when you write the test after implementation... Removing the CI installation and the un-needed mock.

Reply:
Actually the mock needs to be there so that the test performs correctly also locally where `hdf5plugin` is installed. CI environment updated.

Reply:
Now tests are failing...

Reply:
CI just says "Error:" without anything else. I'll restart them and check tomorrow if there's something more available.